### PR TITLE
Added instructions / helpers to run on Canton

### PR DESCRIPTION
**Pull request by daravep**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -130,9 +130,9 @@ class App extends Component {
     if (newMessage.trim() === '') return
 
     const match = /\/(\w+) (.*)/.exec(newMessage)
-    const command = match ? match[1] : 'send'
+    const command = match ? match[1] : (newMessage === '/parties' ? 'parties' : 'send')
```
Comment (davidpadbury-da) [inline src/App.js]: ah - so this is just doing `/parties` alone and in which case the regex isn't matching? perhaps update the expression to `\/(\w+)((?:\s*)(.*))?` and change content to `match[3]`.

File: src/App.js [hunk 2]
```diff
@@ -158,6 +158,14 @@ class App extends Component {
           })
           break;
 
+      case 'parties':
+        const message = this.state.parties.map(x => x.partyId).join("\n")
+        this.setState({
+          messages: this.state.messages.concat({timestamp: new Date(), sender: 'Known Parties', text: message })
```
Comment (davidpadbury-da) [inline src/App.js:164]: I think this message will be blown away the next time a message is sent

shrug

**Review by davidpadbury-da — APPROVED** [review]
nice!

File: README.md [hunk 3]
```diff
@@ -22,6 +22,25 @@ First run `npm install` to install required dependencies.
   - Start the json-api pointed at sandbox
   - Start the React app using the live reloading dev server and open a browser
 
+## Running on Canton
+
+First, run `npm install` to install the dependencies. 
+
+Then, run `npm build` to build the dar.
```
Comment (davidpadbury-da) [inline README.md]: `npm run build`

File: src/App.js (hunk 2 above)
Comment (daravep) [reply src/App.js:164]: Yeah, I know. But that was the only way I could find quickly without substantial changes.

File: src/App.js (hunk 1 above)
Comment (daravep) [reply src/App.js]: fixed

File: README.md (hunk 3 above)
Comment (daravep) [reply README.md]: fixed

**Pull request merged**